.github/workflows/build-binaries-and-brew.yaml

```diff
@@ -67,7 +67,21 @@ jobs:
       # regarding the tiktoken part of the command, see https://github.com/openai/tiktoken/issues/80
       # regarding the litellm part of the command, see https://github.com/pyinstaller/pyinstaller/issues/8620#issuecomment-2186540504
       run: |
-        pyinstaller holmes.py --add-data 'holmes/plugins/runbooks/*:holmes/plugins/runbooks' --add-data 'holmes/plugins/prompts/*:holmes/plugins/prompts' --add-data 'holmes/plugins/toolsets/*:holmes/plugins/toolsets' --hidden-import=tiktoken_ext.openai_public --hidden-import=tiktoken_ext --hiddenimport litellm.llms.tokenizers --hiddenimport litellm.litellm_core_utils.tokenizers --collect-data litellm
+
+        pyinstaller holmes.py \
+          --add-data 'holmes/plugins/runbooks/*:holmes/plugins/runbooks' \
+          --add-data 'holmes/plugins/prompts/*:holmes/plugins/prompts' \
+          --add-data 'holmes/plugins/toolsets/*:holmes/plugins/toolsets' \
+          --add-data 'holmes/plugins/toolsets/coralogix*:holmes/plugins/toolsets/coralogix' \
```

Comment:
Thinking here loud, isn't there a better way to copy all folders recursively. My concern is that we will forget to add the new entries here every time. 

If this isn't supported i think we might consider using another hierarchy. e.g. put all template files in one folder and make sure to copy that. WDYT?

Reply:
I think there are a couple of 'good' and comprehensive fixes:
1. Dynamically generate `--add-data` args by parsing the sources
2. Implement automated smoke tests for the cli

(2) would be extremely beneficial as we'd get notified at PR time.

Reply:
Lets open tech debt for that for next sprint.